CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.8.6 (Next)
+### 0.9.1 (Next)
 
 * Your contribution here.
 
+### 0.9.0 (Unreleased)
+
+* [#257](https://github.com/mongoid/mongoid-history/pull/257): Add track_blank_changes option - [@BrianLMatthews](https://github.com/BrianLMatthews).
```

Comment:
Add a line 

```
* Your contribution here.
```

Reply:
Ok, I'm baffled 😂. Here's what I've done to `CHANGELOG.md` so far:
```
$ git diff master CHANGELOG.md
diff --git a/CHANGELOG.md b/CHANGELOG.md
index b55441d..a5ce98c 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,11 @@
-### 0.8.6 (Next)
+### 0.9.1 (Next)

 * Your contribution here.

+### 0.9.0 (Unreleased)
+
+* [#257](https://github.com/mongoid/mongoid-history/pull/257): Add track_blank_changes option - [@BrianLMatthews](https://github.com/BrianLMatthews).
+
 ### 0.8.5 (2021/09/18)

 * [#250](https://github.com/mongoid/mongoid-history/pull/250): Migrate to Github actions - [@johnnyshields](https://github.com/johnnyshields).
 ```
That seems like what `CONTRIBUTING.md` is asking for, although I guess it doesn't say to renumber the next release, is that the problem? There is a
```
* Your contribution here.
```
line so I don't understand the request to add one. Maybe just tell me what you want it to look like down to the 0.8.5 line, as obviously I'm not getting something.

Reply:
This is what it should look like:

```
### 0.9.0 (Next)

* [#257](https://github.com/mongoid/mongoid-history/pull/257): Add track_blank_changes option - [@BrianLMatthews](https://github.com/BrianLMatthews).
* Your contribution here.
```

Reply:
Ok, thanks, updated it to look like that.